features/api/backend_usages.feature

```diff
@@ -12,17 +12,6 @@ Feature: Backend Usages
     And I go to the provider dashboard
     And I go to the service backends admin page of service "API"
 
-  Scenario: Add a backend api that is unused
```

Comment:
This scenario is being removed in https://github.com/3scale/porta/pull/2910 anyway, there is no point in fixing it here.